mk/spksrc.download.mk

```diff
@@ -63,8 +63,12 @@ download_target: $(PRE_DOWNLOAD_TARGET)
 	      if [ ! -f $${localFile} ]; then \
 	        rm -fr $${localFolder}.part ; \
 	        echo "git clone $${url}" ; \
-	        git clone --no-checkout --quiet $${url} $${localFolder}.part ; \
+	        git clone --no-checkout --recursive --quiet $${url} $${localFolder}.part ; \
 	        git --git-dir=$${localFolder}.part/.git --work-tree=$${localFolder}.part checkout --quiet $(PKG_GIT_HASH) ; \
+	        echo "git submodule update --init --recursive" ; \
```

Comment:
I meant [_your_ L68-L71](https://github.com/wind-rider/spksrc/commit/75c9aec62d066f1185669ffed1962bafad8a739c#diff-61dfff1d55cba7939914513abb791f4fR68-R71), not the lines of existing spksrc code :P You added this:
```
+echo "git submodule update --init --recursive" ; \
+    cd $${localFolder}.part ; \
+    git submodule update --init --recursive --quiet ; \
+    cd .. ; \
```
I'm not sure that does anything worthwhile, because the recursive checkout to cover submodules is done just before that (https://github.com/wind-rider/spksrc/commit/75c9aec62d066f1185669ffed1962bafad8a739c#diff-61dfff1d55cba7939914513abb791f4fR66). There's no need to update the submodules right afterwards. Why did you add it?